components/card.css

```diff
@@ -0,0 +1,109 @@
+/* ECOM CARD */
+.vertical-card {
+	margin: 1rem;
+	display: flex;
+	position: relative;
+	flex-direction: column;
+	width: 25rem;
+	padding: 1rem;
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	-moz-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+}
+
+.card-img img {
+	width: 100%;
+}
+.card-img {
+	width: 100%;
+	position: relative;
+}
+.card-text {
+	font-size: 1.4rem;
+	margin: 1rem;
+}
+.card-btn {
+	display: flex;
+	margin: 1rem 0;
+}
+.card-title {
+	margin: 1rem 0;
+}
+/* CARD PRICE FOR ECOM */
+.card-price-striked {
+	text-decoration: line-through;
+	margin: 1rem;
+	font-size: 1.3rem;
+	color: var(--color-grey-dark);
+}
+.card-price {
+	font-size: 1.6rem;
+	color: var(--color-primary);
+}
+
+/* BLOG CARD */
+.card-container {
+	display: flex;
+	flex-direction: column;
+	margin: 1rem;
+	padding: 1rem;
+	width: 36rem;
+	position: relative;
+	background-color: var(--color-white);
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	-moz-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+}
+.img-box,
+img {
```

Comment:
Repeated styles can be combined lines 14, 58

Also, this can be moved to reset.css

Reply:
Okay